app/views/movies/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1><%= @movie.title %></h1>
+
+<br>Creation Year: <%= @movie.creation_year %>
+<br>Genre: <%= @movie.genre %>
+
+<p><%= form_with url: "/artists", method: :post, data: { turbo: false } do |form| %>
+  <%= form.label :Actor_Submission %>
+  <br><%= form.text_field :actor %>
+  <%= form.submit 'Create Actor' %>
+<% end %></p>
```

Comment:
This form is really close but if we take a closer look at the path and verb that you have here, we see that you have a `post "/artists"` and because we understand REST, this combination would say that we are trying to create a new artist on the `artists` table in the database, which isn't what we're actually trying to do.

We are trying to create a relationship between an ACTOR and a MOVIE, and because we are CREATING this relationship, this would go on a joins table. Because an actor can be in many movies, and a movie can have many actors. So with all of that being said, the route that we want to send this form to is going to be: `post "/movies/#{@movie.id}/actors`.

Here is what the form should look like. Notice I also changed the label and text field as well:

```ruby
<%= form_with url: "/movies/#{@movie.id}/actors", method: :post, data: { turbo: false } do |form| %>
  <%= form.label :actor_id %>
  <br><%= form.text_field :actor_id %>

  <%= form.submit 'Add Actor' %>
<% end %>
```

---

Again, your form is very close but slightly off for what we are trying to do.